### ISSUE TITLE
'>' disapear when going to far in menu

### ISSUE THREAD

**Issue by spilz87** [MarlinFirmware/Marlin#4446]
It's not really important, but sometimes I'm looking for the '>' which disapears when I navigate in menu with coder and when I go too far.
I don't know if it's the good location to report this "issue".

sorry for my English

Spilz

**Comment by jbrazio** [comment]
Thank you @spilz87 for reporting the issue, we're looking into it.

**Comment by thinkyhead** [comment]
Probably a simple bug. I'll mess with my LCD tonight and come up with a patch.

**Comment by thinkyhead** [comment]
@spilz87 I believe this might fix it for you. Add the following new line in `ultralcd.cpp`:

``` diff
// Scroll through menu items, scrolling as-needed to stay in view
#define END_MENU() \
  } \
  if (encoderLine >= _thisItemNr) { \
    encoderLine = _thisItemNr - 1; \
    encoderPosition = encoderLine * (ENCODER_STEPS_PER_MENU_ITEM); \
+   lcdDrawUpdate = LCDVIEW_CALL_REDRAW_NEXT; \
  } \
  if (encoderLine >= encoderTopLine + LCD_HEIGHT) { \
    encoderTopLine = encoderLine - (LCD_HEIGHT - 1); \
    lcdDrawUpdate = LCDVIEW_CALL_REDRAW_NEXT; \
  } \
  UNUSED(_skipStatic)
```

**Comment by spilz87** [comment]
Thanks,
I will test it when I come back home, end of week

**Comment by boelle** [comment]
@spilz87 did you test this?

**Comment by jbrazio** [comment]
Closed by #4512.
Pleas reopen this issue if the problem still persists.

**Comment by spilz87** [comment]
I just test it, it seems ok. 
I didn't try all menu

Thanks for your help
